### PR TITLE
Add docs and fix unsafe casts

### PR DESCRIPTION
**Pull request by yk-sgr**

### CODE REVIEW

**Comment by yk-sgr** [comment]
If someone wants to add better docs, go for it!

**Comment by DRSchlaubi** [comment]
Going to review it when it passes the checks

**Comment by yk-sgr** [comment]
Checks passed.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # VoteBot
 [![Build Status](https://travis-ci.org/Votebot/voteBot.svg?branch=master)](https://travis-ci.org/Votebot/voteBot)
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/a9936806f3cf4f77921e91d9d9eb5fbb)](https://www.codacy.com/app/VoteBot/voteBot?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=Votebot/voteBot&amp;utm_campaign=Badge_Grade)
```
Comment (DRSchlaubi) [inline README.md:3]: Didn't you already add that in #12

Comment (yk-sgr) [reply README.md:3]: Forgot to checkout master branch.

**Review by DRSchlaubi — APPROVED** [review]

**Pull request merged**